labware-library/src/labware-creator/components/alerts/StackingAlerts.tsx

```diff
@@ -0,0 +1,25 @@
+import { AlertItem } from '@opentrons/components'
+
+import { MAX_SUGGESTED_GRIPPER_Z } from '../../fields'
+
+import type { FormikTouched } from 'formik'
+import type { LabwareFields } from '../../fields'
+
+export interface Props {
+  values: LabwareFields
+  touched: FormikTouched<LabwareFields>
+}
+
+export const StackingAlerts = (props: Props): JSX.Element | null => {
+  const { values, touched } = props
+  const zAsNum = Number(values.stackedLabwareZDimension)
```

Comment:
```suggestion
  const zAsNum = values.stackedLabwareZDimension ?? 0
```

An `undefined` value coerced to be a `Number` yields `NaN`, which we probably don't want to handle. I think we want to nullish coalesce to a real value here. I'm guessing 0 is what we want?